packages/components/molecules/f-skeleton-loader/src/components/skeletons/RestaurantCarouselCard.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div :class="$style['c-restaurantCarouselCard']">
+        <carousel-card-background :class="$style['c-restaurantCarouselCard-background']" />
```

Comment:
Can you use Image.vue component here straight away? It feels like `carousel-card-background` component just wraps image component in a div and adds `height: 200px;` to it which can be done here if I haven't missed anything.

Reply:
Perfect thanks